src/classes/STG_InstallScript.cls

```diff
@@ -369,6 +377,21 @@ global without sharing class STG_InstallScript implements InstallHandler {
         }
     }
 
+    /*******************************************************************************************************
+    * @description Creates the finalized opp stage to state record based on what value is set as the 
+    * "Opportunity Stage for When Fully Paid" 
+    */
+    private void createOppStageToStateRecordsIfNecessary() {
+
+        List<Metadata.CustomMetadata> metadataToInsert = new List<Metadata.CustomMetadata>();
+        Map<String, Object> params = new Map<String, Object>();
+
+        metadataToInsert = Opp_StageMappingUtil.createFinalizedOppStageToStateRecord(metadataToInsert);
+        metadataToInsert = Opp_StageMappingUtil.createPledgedOppStageToStateRecord(metadataToInsert);
```

Comment:
Given the requirements, you will probably need to swap these.  Also, ideally, it should only add the committed mapping if the Pledged Stage exists in the OpportunityStage table.

Reply:
I'll switch the order of these two. The check for the pledge stage is done inside of createPledgedOppStageToStateRecord.

Reply:
And again, it should not check for Pledge Stage.  Just whether the "Pledged Stage" exists, and whether there are any metadata records yet.